atomist/configure/settings.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Settings
+description:
+keywords:
+---
+
+This page describes the configurable settings for Atomist. Enabling any of these
+settings instructs Atomist to carry out an action whenever a specific git event
+occurs. These features require that you
+[install the Atomist GitHub app](/atomist/integrate/github/#connect-to-github){:
+target="blank" rel="noopener" class=""} in your GitHub organization.
+
+To view and manage these settings, go to the
+[settings page](https://dso.docker.com/r/auth/policies){: target="blank"
+rel="noopener" class=""} on the Atomist website.
+
+## New image vulnerabilities
+
+Scan container images for new critical and high-severity vulnerabilities
```

Comment:
We should avoid using _scan_ and position us as _extract SBOM_ and _match packages against advisories_.